MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -15,54 +15,63 @@ final class MovieQuizViewController: UIViewController {
     @IBOutlet private weak var noButton: UIButton!
 
     // MARK: - Properties
-    
-    private var currentQuestionIndex: Int = 0
-    private var gamesScore: QuizScores = QuizScores()
-    
-    private let questionsAmount: Int = 10
-    private let questionFactory: QuestionFactoryProtocol = QuestionFactory()
-    private var currentQuestion: QuizQuestion?
-    
-    private var resultAlertPresenter: ResultAlertPresenter? = nil
+
+    private var presenter: MovieQuizPresenter! = nil
```

Comment:
Можете объяснить почему решили использовать Implicitly Unwrapped Optional?
Если можете, то предлагаю убрать лишний nil: 
```suggestion
    private var presenter: MovieQuizPresenter!
```
Иначе, давайте использовать опционалы, так вы убережете себя от ошибок сразу:
```suggestion
    private var presenter: MovieQuizPresenter?
```